apps/src/hamburger/hamburger.js

```diff
@@ -40,5 +44,26 @@ export const initHamburger = function () {
       }
     });
 
+    $("#hamburger #report-bug").click(function () {
+      trackEvent("help_ui", "report-bug", "hamburger");
+    });
+
+    $("#hamburger #support").click(function () {
+      trackEvent("help_ui", "support", "hamburger");
+    });
+
+    // This item is not in the hamburger, but actually in the studio footer.
+    $(".footer #support").click(function () {
+      trackEvent("help_ui", "support", "studio_footer");
+    });
+
+    // This item is not in the hamburger, but actually in the pegasus footers for
+    // desktop and mobile.
+    $("#pagefooter #support").each(function () {
+      $(this).click(function () {
+        trackEvent("help_ui", "support", "studio_footer");
+      });
+    });
+
```

Comment:
@Bjvanminnen It's not ideal to use `hamburger.js` for events that are actually attached to the footers, but it seemed useful to keep all this code together, and we know that `hamburger.js` is properly-pipelined JS code that ends up on all the right pages.  Is there a better place to put this code?